pkg/cleaner/cleaning_test.go

```diff
@@ -90,7 +89,7 @@ func TestObfuscateReaderIOErrorPropagates(t *testing.T) {
 	require.ErrorIs(t, err, UnwritableErr)
 }
 
-func TestObfuscateFileOutputExistsFails(t *testing.T) {
+func TestObfuscateFileOutputExists(t *testing.T) {
```

Comment:
can you add another test that will check that the files are created with the right pattern (ascending numbers appended)?

Reply:
Sure